### PR TITLE
add license statement for PEAR

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by nsoranzo** [comment]
@bgruening I've cancelled the `push` build to save some Travis time. Can you open PR from your fork next time? Thanks!!

**Comment by bgruening** [comment]
Can we simply disable them in travis?

File: tools/pear/pear.xml [hunk 1]
```diff
@@ -159,6 +159,9 @@ For more information please look at the documentation_ and `github repository`_.
 .. _documentation: http://sco.h-its.org/exelixis/web/software/pear/doc.html
 .. _github repository: https://github.com/xflouris/PEAR
```
Comment (nsoranzo) [inline tools/pear/pear.xml]: The GitHub repo has disappeared, you can probably remove it from help.

**Comment by nsoranzo** [comment]
> Can we simply disable them in travis?

No, you can only enable/disable "Build branch updates", which means that we would disable also builds on `master` (the ones that push to TTS and MTS).

**Pull request merged**